generators/entity-client/partials/react/entity-detail.tsx.js

```diff
@@ -0,0 +1,29 @@
+const file = context => `${context.webappDir}app/entities/${context.entityFolderName}/${context.entityFileName}-detail.tsx`;
+
+const tmpls = [
+    {
+        type: 'replaceContent',
+        regex: true,
+        target: context => `<Translate(.*)\n(\\s*)${context.tenantNameUpperFirst}\n(\\s*)</Translate>`, // eslint-disable-next-line
+        tmpl: context => `{!this.props.account.${context.tenantNameLowerFirst} ? <Translate contentKey="${context.baseName}App.car.${context.tenantNameLowerFirst}">
+        ${context.tenantNameUpperFirst}</Translate> : ''}`
+    },
+    {
+        type: 'replaceContent',
```

Comment:
Note sure exactly what the 2 updates below are for? But Im not that familiar with the React codebase.